test_assets/CompositionPuzzles/PayloadAndReference/README.md

```diff
@@ -0,0 +1,192 @@
+# Payload and reference ordering
+
+![problem](images/Slide108.PNG)
+
+This puzzle is inspired by a real-life USD asset structure problem at encountered during USD adoption at a large animation studio. 
```

Comment:
small grammar suggestion:

This puzzle was inspired by a real-life USD asset structure problem encountered during USD adoption at a large animation studio.